src/components/Background/index.css

```diff
@@ -1,9 +1,15 @@
 @import "../../styles/colors.css";
+@import "../../styles/layout.css";
 
 .root {
   position: relative;
 }
 
+.max-width {
```

Comment:
Name this `horizontalSpacing` and change the prop, this should have horizontal padding instead of max width. The space on each side is $gutter and $gutter/2 in mobile